WireAnalytics/Sources/WireAnalytics/Countly/Countly+CountlyProtocol.swift

```diff
@@ -18,40 +18,39 @@
 
 import Countly
 
-extension Countly: AnalyticsService {
+extension Countly: CountlyProtocol {
```

Comment:
Previously `AnalyticsService` was an abstraction around `Countly`, now this is done by `CountlyProtocol`.